acceptance_test.go

```diff
@@ -0,0 +1,101 @@
+// Copyright © 2024 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package activemq
+
+import (
+	"fmt"
+	"math/rand"
+	"testing"
+	"time"
+
+	sdk "github.com/conduitio/conduit-connector-sdk"
+)
+
+func TestAcceptance(t *testing.T) {
+	cfg := map[string]string{
+		"url":      "localhost:61613",
+		"user":     "admin",
+		"password": "admin",
+	}
+
+	driver := sdk.ConfigurableAcceptanceTestDriver{
+		Config: sdk.ConfigurableAcceptanceTestDriverConfig{
+			Connector:         Connector,
+			SourceConfig:      cfg,
+			DestinationConfig: cfg,
+			BeforeTest: func(t *testing.T) {
```

Comment:
Maybe delete them in the AfterTest?

Reply:
I don't think it's necessary. After all, the kafka connector acceptance tests uses an uuid to make the topic names unique. I am doing the same here.

I'll remove the comment, it is stale.